config/dev.webpack.config.js

```diff
@@ -17,6 +17,49 @@ const calculateRemoteConfig = (remoteConfig) => {
   return `https://${remoteConfig}.console.redhat.com`;
 };
 
+const kcpProxy = [
+  {
+    context: (path) => path.includes('/api/k8s'),
+    target: process.env.KCP_SERVER,
+    secure: false,
+    changeOrigin: true,
+    autoRewrite: true,
+    ws: true,
+    pathRewrite: { '^/api/k8s': '' },
+  },
+  {
+    context: (path) => path.includes('/wss/k8s'),
+    target: process.env.KCP_SERVER.replace('https://', 'wss://'),
```

Comment:
This line throws an error when running `npm run start:prod:beta` because `process.env.KCP_SERVER` is `undefined`.